Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2.yml

```diff
@@ -188,141 +136,166 @@ tasks:
-      playbookName: FireEye HX - Isolate Endpoint
-      type: playbook
-      version: -1
-    taskid: 48226108-0787-44b0-80f6-cf333758b5e8
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1340,
+          "y": 195
+        }
+      }
+    note: false
     timertriggers: []
-    type: playbook
-    view: "{\n  \"position\": {\n    \"x\": 1340,\n    \"y\": 195\n  }\n}"
-  '11':
-    id: '11'
     ignoreworker: false
-    isautoswitchedtoquietmode: false
+    skipunavailable: true
+    quietmode: 0
     isoversize: false
-    loop:
-      exitCondition: ''
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 04a17761-3b84-492f-87c6-ef29fc4adc7d
+    type: playbook
+    task:
+      id: 04a17761-3b84-492f-87c6-ef29fc4adc7d
+      version: -1
+      name: Block Endpoint - Carbon Black Response V2
+      description: Carbon Black Response - isolate an endpoint for a given hostname.
+      playbookName: Block Endpoint - Carbon Black Response V2
+      type: playbook
       iscommand: false
-      max: 100
-      wait: 1
+      brand: ""
     nexttasks:
       '#none#':
-      - '2'
-    note: false
-    quietmode: 0
+      - "2"
     scriptarguments:
       Hostname:
         simple: ${inputs.Endpoint_hostname}
       Sensor_id:
         simple: ${inputs.Endpoint_id}
     separatecontext: true
-    skipunavailable: true
-    task:
-      brand: ''
-      description: Carbon Black Response - isolate an endpoint for a given hostname.
-      id: 04a17761-3b84-492f-87c6-ef29fc4adc7d
+    continueonerrortype: ""
+    loop:
       iscommand: false
-      name: Block Endpoint - Carbon Black Response V2
-      playbookName: Block Endpoint - Carbon Black Response V2
-      type: playbook
-      version: -1
-    taskid: 04a17761-3b84-492f-87c6-ef29fc4adc7d
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1770,
+          "y": 195
+        }
+      }
+    note: false
     timertriggers: []
-    type: playbook
-    view: "{\n  \"position\": {\n    \"x\": 1770,\n    \"y\": 195\n  }\n}"
-  '12':
-    id: '12'
     ignoreworker: false
-    isautoswitchedtoquietmode: false
+    skipunavailable: true
+    quietmode: 0
     isoversize: false
-    loop:
-      exitCondition: ''
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: d438379a-5602-49f9-8d6d-404f9dbe3919
+    type: playbook
+    task:
+      id: d438379a-5602-49f9-8d6d-404f9dbe3919
+      version: -1
+      name: Microsoft Defender For Endpoint - Isolate Endpoint
+      playbookId: Microsoft Defender For Endpoint - Isolate Endpoint
+      type: playbook
       iscommand: false
-      max: 100
-      wait: 1
+      brand: ""
+      description: ''
     nexttasks:
       '#none#':
-      - '2'
-    note: false
-    quietmode: 0
+      - "2"
     scriptarguments:
       Device_IP:
         simple: ${inputs.Endpoint_ip}
```

Comment:
Can we already update the inputs entered manually to complex?

Reply:
fixed